content/static/doc/tutorial/getting-started.html

```diff
@@ -157,57 +141,45 @@ <h2 id="code">Write some code</h2>
   </li>
 </ol>
 
-<h2 id="call">Call code in an external package</h2>
+<h2 id="call">使用外部包的代码</h2>
 
 <p>
-  When you need your code to do something that might have been implemented by
-  someone else, you can look for a package that has functions you can use in
-  your code.
+  当你当代码需要引用其他代码的时候，你可以查找其他包中包含你想要的功能。
 </p>
 
 <ol>
   <li>
-    Make your printed message a little more interesting with a function from an
-    external module.
-
+    通过使用其他模块中的功能使你的打印信息更加有趣。
     <ol>
       <li>
-        Visit pkg.go.dev and
+        访问 pkg.go.dev 并且
         <a href="https://pkg.go.dev/search?q=quote"
-          >search for a "quote" package</a
+          >查找 "quote" 包</a
         >.
       </li>
       <li>
-        Locate and click the <code>rsc.io/quote</code> package in search results
-        (if you see <code>rsc.io/quote/v3</code>, ignore it for now).
+        在搜索结果中找到并点击 <code>rsc.io/quote</code> 包（
+        当你看到  <code>rsc.io/quote/v3</code> 包时请暂时忽略）。、
       </li>
       <li>
-        On the <strong>Doc</strong> tab, under <strong>Index</strong>, note the
-        list of functions you can call from your code. You'll use the
-        <code>Go</code> function.
+        在 <strong>文档</strong> 标签，索引里面，注意列表里面是你可以调用的功能。你可以使用 <code>Go</code> 函数。
```

Comment:
```suggestion
        在左侧 <strong>Documentation(文档)</strong> 标签页面的 Index(索引) 下面的清单里列举的是你可以调用的功能（function）。你将使用的 <code>Go语言</code> 函数。
```

这里描述的是 https://pkg.go.dev/ 的操作指引，不应该直译。